src/core/chunked_stream.js

```diff
@@ -35,9 +35,16 @@ var createPromiseCapability = sharedUtil.createPromiseCapability;
 var isInt = sharedUtil.isInt;
 var isEmptyObj = sharedUtil.isEmptyObj;
 
-var ChunkedStream = (function ChunkedStreamClosure() {
-  function ChunkedStream(length, chunkSize, manager) {
-    this.bytes = new Uint8Array(length);
+/* If the length of stream is less than ALLOCATE_NO_CHUNKS_SIZE then use
+ChunkedStreamContinuous which works with single block of memory */
+var ALLOCATE_NO_CHUNKS_SIZE = 4 * 1024 * 1024;
+
+/* Base abstract class for Continuous and Fragmented versions
+of chunked stream. Loads PDF data in chunks or by progressive downloads */
```

Comment:
Nit: for comments that applies to "classes" and function, please use the following style instead:
```javascript
/**
 * Base abstract class for Continuous and Fragmented versions
 * of chunked stream. Loads PDF data in chunks or by progressive downloads.
 */
```